url_shortner.py

```diff
@@ -14,27 +15,26 @@
 Base = declarative_base()
 
 
-class SQLiteBackend(object):
-    """ SQLite Backend that manages 
+class PostgresBackend(object):
```

Comment:
Change this to SQLBackend and update the references everywhere else.

Basically using SQL Alchemy, it should work for any database, sql alchemy handles things transparently according to each database because you see, there were not much changes in code when using Postgres from sqlite.

Reply:
change made